### ISSUE TITLE
Enhanced dependencies for Swedish

### ISSUE THREAD

**Issue by LarsAhrenberg** [UniversalDependencies/docs#768]
I want to add extended dependencies to UD_Swedish-Lines and have come across some phenomena where the guidelines are not specific enough or inconsistent with the annotation in UD_Swedish-Talbanken.

1. Swedish has a considerable number of sentences where a subordinate clause is introduced by ADP + SCONJ, both independently assigned 'mark' as deprel. Example:

`Genom att överföra det ekonomiska ansvaret på kibbutzen ... `

> 1	Genom	genom	ADP	PP	_	3	mark	3:mark	_
> 2	att	att	PART	IE	_	3	mark	3:mark	_
> 3	överföra	överföra	VERB	VB|INF|AKT	VerbForm=Inf|Voice=Act	13	advcl	13:advcl:att	_
> ...

My question is why the ADP 'genom' is not attached to the enhanced dependency, giving the annotation `13:advcl:genom:att`. After all, the rule says 'We therefore augment nmod, obl, acl and advcl relation labels with the preposition or the case of the modifier'.

2.  When a relative clause is preceded by a nominal coordination it could attach only to the last conjunct or to all of them. The following example illustrates the first case, but I would argue that the other interpretation is equally plausible: 
`Flera vetenskapsmän har gjort det, särskilt sociologer och psykologer, som i kibbutzerna sett ...`

> 8	       sociologer	sociolog	NOUN	...  4	parataxis	4:parataxis	_
> 9	       och	och	CCONJ	KN	_	10	cc	10:cc	_
> 10	psykologer	psykolog	NOUN	...  8	conj	8:conj:och|15:nsubj	SpaceAfter=No
> 11	,	,	PUNCT	MID	_	15	punct	15:punct	_
> 12	som	som	PRON	HP|-|-|-	PronType=Rel	15	nsubj	10:ref	_
> 13	i	i	ADP	PP	_	14	case	14:case	_
> 14	kibbutzerna	kibbutz	NOUN	... 15	obl	15:obl:i	_
> 15	sett	se	VERB	...  10	acl:relcl	10:acl:relcl	_
> 

If so, it is clear that also token 8, sociologer, should receive '15:nsubj' as an added enhanced dependency. But should anything be added to the relative pronoun _som_? Should it be 8:ref|10:ref or just 8:ref?

**Comment by perrier54** [comment]
In the first case of multiple mark relations, I agree with @LarsAhrenberg.

In the second case, there is first an error in the basic UD annotation: the source of the `acl:relcl` relation must be the head of the coordination, that is `sociologer`. At the enhanced level, the relation becomes `8:acl:relcl | 10:acl:relcl` because the governors are systematically distributed on the conjuncts according to the guidelines: _" The enhanced representation therefore also contains dependencies between the other conjuncts and the governor and dependents of the phrase."_

It is the same for the relation `ref`. We  must have `8:ref | 10:ref`.

The rule also works for the propagation of dependents and  I agree that `sociologer `should receive `15:nsubj`.

**Comment by jnivre** [comment]
I think I agree with both of you, especially on the second point. There is clearly an error in the basic annotation, which has misled the automatic enhancer. We have manually checked a subset of the enhancement, notably added subject relations in xcomp constructions and added null nodes in elliptical constructions, but the conjunct propagation has not been checked at all so far.

On the first point, Lars is right that the guidelines are simply vague, but I agree that it makes sense to concatenate multiple mark dependents (and similarly for case). But this is not a functionality that is implemented in the Stanford enhancer, which is what we have used to add these enhancements, which are otherwise deterministic.
